STM32/SPI/Src/ADS1120.c

```diff
@@ -96,13 +96,56 @@ static bool isDataReady(ADS1120Device *dev)
     return !stmGetGpio(dev->drdy);
 }
 
-//static void powerDown(SPI_HandleTypeDef *hspi);     // Enter power-down mode (not implemented)
+// it is possible to a single register but to simplify stuff, just read it all.
+static HAL_StatusTypeDef readRegisters(ADS1120Device *dev, ADS1120_RegConfig* regs)
```

Comment:
Nit-pick: Rather than moving this function around, wouldn't prototypes be better?

I mainly say this because it means you can a) move static functions to the bottom of the file, where they belong and b) it means that when one reviews it, one doesn't waste time double checking in the  diff if it has actually changed